skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -140,6 +140,9 @@ actual open class SkiaLayer internal constructor(
 
     val windowHandle: Long
         get() = backedLayer.windowHandle
+    
+    val currentDPI: Long
```

Comment:
Please add docs for new public fields and methods (and documenting nearby fields, unless documented in expect declaration) would be neat.

Reply:
Done.